sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -251,3 +253,18 @@ def __init__(self):
                         check_no_fit_attributes_set_in_init,
                         'estimator_name',
                         NonConformantEstimator)
+
+
+def test_check_estimator_pairwise():
+    # check that check_estimator() works on estimator with _pairwise
+    # attribute set
+    est = SVC(kernel='precomputed')
+    check_estimator(est)
+
+
+def test_check_estimator_metric_and_kernel():
```

Comment:
There's no kernel here. But you also need metric=precomputed for this to pertain.

Perhaps to make sure these tests are doing what they're meant to, you should assert that the estimator is pairwise